tests/e2e/buildah_suite_test.go

```diff
@@ -0,0 +1,337 @@
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"encoding/json"
+	"github.com/containers/image/copy"
+	"github.com/containers/image/signature"
+	"github.com/containers/image/storage"
+	"github.com/containers/image/transports/alltransports"
+	"github.com/containers/image/types"
+	sstorage "github.com/containers/storage"
+	"github.com/containers/storage/pkg/reexec"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+)
+
+var (
+	PODMAN_BINARY      string
+	CONMON_BINARY      string
+	CNI_CONFIG_DIR     string
+	RUNC_BINARY        string
+	INTEGRATION_ROOT   string
+	STORAGE_OPTIONS    = "--storage-driver vfs"
+	ARTIFACT_DIR       = "/tmp/.artifacts"
+	CACHE_IMAGES       = []string{"alpine", "busybox", fedoraMinimal}
+	RESTORE_IMAGES     = []string{"alpine", "busybox"}
+	ALPINE             = "docker.io/library/alpine:latest"
+	BB_GLIBC           = "docker.io/library/busybox:glibc"
+	fedoraMinimal      = "registry.fedoraproject.org/fedora-minimal:latest"
+	defaultWaitTimeout = 90
+)
+
+// BuildAhSession wrapps the gexec.session so we can extend it
+type BuildAhSession struct {
+	*gexec.Session
+}
+
+// BuildAhTest struct for command line options
```

Comment:
OK, I'm giggling everytime I see BuildaAh*, nicely done sir!

Reply:
Yeah, I don't really get that.  There's a few stray references to podman in here, too?

Reply:
ill correct the strays